src/sangeeta/ShopWithScanner.java

```diff
@@ -0,0 +1,29 @@
+package sangeeta;
+
+import java.util.Scanner;
+
+public class ShopWithScanner {
+
+	public static void main(String[] a) {
+		Scanner scanner = new Scanner(System.in);
+		int MaggieQ = 50;
+		int PanipuriQ = 60;
+		System.out.println("Available Quantity of Maggie: " + MaggieQ);
+		System.out.println("Available Quantity of Panipuri: " + PanipuriQ);
+		System.out.println("Enter quantity of Maggie & Panipuri you want: ");
+		int MaggieR = scanner.nextInt();
+		int PanipuriR = scanner.nextInt();
+		if (MaggieQ >= MaggieR) {
```

Comment:
this is not right way to implement 

please refer below code

Madhavi R - https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/564/files
Dipali - https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/559/files